src/utils/openPopup.js

```diff
@@ -20,48 +20,60 @@ export const openPopup = (params) => {
   const container = dom.getContainer()
   const popup = dom.getPopup()
 
-  if (params.onBeforeOpen !== null && typeof params.onBeforeOpen === 'function') {
+  if (typeof params.onBeforeOpen === 'function') {
     params.onBeforeOpen(popup)
   }
 
-  if (params.animation) {
-    dom.addClass(popup, swalClasses.show)
-    dom.addClass(container, swalClasses.fade)
-  }
-  dom.show(popup)
+  addClasses({ container, popup })(params)
 
   // scrolling is 'hidden' until animation is done, after that 'auto'
+  setScrollingVisibility({ container, popup })
```

Comment:
please use the list of arguments instead of the object of arguments, the same as in https://github.com/sweetalert2/sweetalert2/pull/1641#discussion_r300373805

Reply:
You are right, I will do that